src/wordpress/integration.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Integration interface definition.
+ *
+ * @package Yoast\YoastSEO\WordPress
+ */
+
+namespace Yoast\YoastSEO\WordPress;
+
+/**
+ * An interface for registering integrations with WordPress
+ */
+interface Integration {
```

Comment:
Same as for the `Integration_Group`. I would introduce these concepts to the plugin ASAP.